packages/nuxi/src/utils/env.ts

```diff
@@ -1,7 +1,7 @@
 export const overrideEnv = (targetEnv: string) => {
   const currentEnv = process.env.NODE_ENV
   if (currentEnv && currentEnv !== targetEnv) {
-    console.warn(`Changing \`NODE_ENV\` from \`${currentEnv}\` to \`${targetEnv}\`.`)
+    console.warn(`Defined \`NODE_ENV\` will be replaced from \`${currentEnv}\` to \`${targetEnv}\`, to avoid unintended behaviours from libaries in development mode.`)
```

Comment:
(we also do this for prod)
```suggestion
    console.warn(`Defined \`NODE_ENV\` will be replaced from \`${currentEnv}\` to \`${targetEnv}\`, to avoid unintended behaviors from libraries.`)
```